src/features/MovieDetails/styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    max-width: 1920px;
+    background: ${({ theme }) => theme.colors.site.backgroundWrapper};
+    padding: 64px;
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.medium}px) {
+    padding: 16px;
+    };
+`;
```

Comment:
This wrapper is probably unnecessary, but you probably added it for testing.

Reply:
on this stage of app its giving this platinum area between edges/header and main part of MovieDetails feature:).
I added  "margin 0 auto" and on this  branch the header and "main" is looking perfect (almost;))